bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -156,16 +154,17 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
             call.caller_address,
             AccountField::Balance,
             caller_balance,
-            caller_balance,
         );
 
-        // Transfer value only for CALL opcode, insufficient_balance = false
-        // and value > 0.
-        if call.kind == CallKind::Call && !insufficient_balance && !call.value.is_zero() {
+        // TODO: What about transfer for CALLCODE?
```

Comment:
When encountering `CALLCODE` the account should be deployed or deploying itself, for the the first case the account exists, and the latter the account is also created after this PR, so it seems correct to only do the transfer when it's `CALL`.